ui/app/components/send_/send-content/send-to-row/send-to-row.component.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import SendRowWrapper from '../send-row-wrapper/'
+import EnsInput from '../../../ens-input'
+
+export default class SendToRow extends Component {
+
+  static propTypes = {
+    closeToDropdown: PropTypes.func,
+    inError: PropTypes.bool,
+    network: PropTypes.string,
+    openToDropdown: PropTypes.func,
+    to: PropTypes.string,
+    toAccounts: PropTypes.array,
+    toDropdownOpen: PropTypes.bool,
+    updateSendTo: PropTypes.func,
+    updateSendToError: PropTypes.func,
+  };
+
+  handleToChange (to, nickname = '') {
+    const { updateSendTo, updateSendToError } = this.props
+    updateSendTo(to, nickname)
+    updateSendToError(to)
+  }
+
+  render () {
+    const {
+      closeToDropdown,
+      inError,
+      network,
+      openToDropdown,
+      to,
+      toAccounts,
+      toDropdownOpen,
+    } = this.props
+
+    return (
+      <SendRowWrapper
+        errorType={'to'}
+        label={`${this.context.t('to')}:`}
```

Comment:
Can you remove the ":" in `"to"` in `messages.json`? Otherwise there will be double colons:
![image](https://user-images.githubusercontent.com/8051479/40329800-a1a97346-5cff-11e8-9812-bd654cf835fb.png)